html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+<h1>Coding Form!</h1>
+<form>
```

Comment:
You should not have an individual `form` element around each input, there should be one `form` element surrounding all the contents of that form.